viewing_party/main.py

```diff
@@ -0,0 +1,151 @@
+#Wave 1 
+def create_movie(title,genre,rating): 
+    if (title and genre and rating): 
+        movie = {}
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie
```

Comment:
I'd recommend building the dictionary with all the attributes instead of creating an empty one and adding values:

```suggestion
        return {
            "title": title,
            "genre": genre,
            "rating": rating
        }
```

It's just a little less typing and I find it a little easier to read.  Plus you don't have to come up with a name for a variable.  😄